packit_service/models.py

```diff
@@ -213,33 +197,33 @@ def get_project_prs(
         cls, first: int, last: int, forge: str, namespace: str, repo_name: str
     ) -> Optional[Iterable["PullRequestModel"]]:
         with get_sa_session() as session:
-            project = cls.__choose_project(
-                session=session, forge=forge, namespace=namespace, repo_name=repo_name
-            )
-            if not project:
-                return None
             pull_requests = (
                 session.query(PullRequestModel)
-                .filter_by(project_id=project.id)
+                .join(GitProjectModel)
+                .filter(
+                    PullRequestModel.project_id == GitProjectModel.id,
+                    GitProjectModel.instance_url == forge,
+                    GitProjectModel.namespace == namespace,
+                    GitProjectModel.repo_name == repo_name,
+                )
                 .order_by(desc(PullRequestModel.pr_id))[first:last]
```

Comment:
What is the difference between the filtering `PullRequestModel` by `project_id` and this join?

Reply:
In the previous implementation, you required to query a project first.
Here you are getting a list of PR's direct without the need for GitProjectModel instance.

Reply:
OK, just curious...